doc/change-project-tags.md

```diff
@@ -0,0 +1,23 @@
+Change project tags
+===================
+
+For [`YCM`-based superbuilds](http://robotology.github.io/ycm/gh-pages/git-master/manual/ycm-superbuild.7.html), the specific 
+branch, tag or commit used for each specific CMake package can be manually specified  using the [`<package>_TAG`](http://robotology.github.io/ycm/gh-pages/git-master/manual/ycm-superbuild.7.html#overriding-parameters) 
+CMake variables. By default, the `robotology-superbuild` uses the latest "stable" branches of the robotology repositories, but in some cases it may be necessary to use the "unstable" active development branches, 
+or use some fixed tags. These advanced use cases are supported by the `ROBOTOLOGY_PROJECT_TAGS` CMake variable. 
+
+`ROBOTOLOGY_PROJECT_TAGS` is an advanced CMake option, so it is visible from the CMake GUI only after enabling the visualization of advanced variables.
+It can take one of three possible values: 
+* `Stable` :  this is the default value, that will use the tags or branches specified in [`cmake/ProjectTagsStable.cmake`](cmake/ProjectTagsStable.cmake) 
+              that are the stables branches for the robotology projects.
+* `Unstable` : by selecting this option, you will use the tags or branches specified in [`cmake/ProjectTagsUnstable.cmake`](cmake/ProjectTagsUnstable.cmake), 
+               that are the "unstable" active development branches for the robotology projects. This is reccomended just for users that work at the IIT labs in Genoa,
+               as it can break compilation or runtime behaviour without any notice.
+* `Custom` : by selecting this option, you need to manually specify the tags or branches that you want to use by specifying a custom project tags file in the 
+             `ROBOTOLOGY_PROJECT_TAGS_CUSTOM_FILE` CMake option. This is useful when  you want to use a fixed version of the software built by the superbuild.
```

Comment:
If I understood the logic, I would also write that this file can be either kept outside the superbuild source directory by specifying an absolute path, or relative and without the `.txt` extension if stored in the `cmake/` folder (and maybe commit it in a temporarily fork / branch).